app/jobs/content_depositor_change_event_job.rb

```diff
@@ -17,26 +17,26 @@ def initialize(id, login, reset=false)
 
   def run
     # TODO: This should be in its own job, not this event job
-    file = ::GenericFile.find(id)
-    file.proxy_depositor = file.depositor
-    file.clear_permissions! if reset
-    file.apply_depositor_metadata(login)
-    file.save!
+    work = ::GenericWork.find(id)
+    work.proxy_depositor = work.depositor
+    work.clear_permissions! if reset
+    work.apply_depositor_metadata(login)
+    work.save!
```

Comment:
Looking through it, I don't believe I am.  I guess a work and the contained files having different depositors wasn't causing conflicts.  The attribute of proxy_depositor comes from the ProxyDeposit concern which GenericFiles no longer have.

I guess I could loop over each of the generic files of the generic work and apply_depositor_metadata.